pcsx2/GS/GSCrc.cpp

```diff
@@ -314,36 +317,33 @@ const CRC::Game CRC::m_games[] =
 	{0x812C5A96, ShinOnimusha, EU, 0},
 	{0xFE44479E, ShinOnimusha, US, 0},
 	{0xFFDE85E9, ShinOnimusha, US, 0},
-	{0xE21404E2, GetaWay, US, TextureInsideRt},
-	{0xE8249852, GetaWay, JP, TextureInsideRt},
-	{0x458485EF, GetaWay, EU, TextureInsideRt},
-	{0x5DFBE144, GetaWay, EU, TextureInsideRt},
-	{0xE78971DF, GetaWayBlackMonday, US, TextureInsideRt},
-	{0x342D97FA, GetaWayBlackMonday, US, TextureInsideRt}, // Demo
-	{0xE8C0AD1A, GetaWayBlackMonday, JP, TextureInsideRt},
-	{0x09C3DF79, GetaWayBlackMonday, EU, TextureInsideRt},
-	{0x1130BF23, SakuraTaisen, CH, 0}, // cutie comment
+	{0xE21404E2, GetaWayGames, US, TextureInsideRt}, // GetaWay
```

Comment:
Minor nitpick: Can you rename them to GetAwayGames?

Reply:
Even that's wrong, it's Getaway